packages/flutter/lib/src/material/list_item.dart

```diff
@@ -5,53 +5,123 @@
 import 'package:flutter/widgets.dart';
 
 import 'ink_well.dart';
+import 'theme.dart';
 
+/// Material List items are one to three lines of text optionally flanked by icons.
+/// Icons are defined with the [left] and [right] parameters. The first line of text
+/// is not optional and is specified with [primary]. The value of [secondary] will
+/// occupy the space allocated for an aditional line of text, or two lines if
+/// isThreeLine: true is specified. If isDense: true is specified then the overall
+/// height of this list item and the size of the DefaultTextStyles that wrap
+/// the [primary] and [secondary] widget are reduced.
 class ListItem extends StatelessComponent {
   ListItem({
     Key key,
     this.left,
```

Comment:
I suppose that when we add support for RTL layouts there will be many left and rights to contend with. Toolbar, margins, padding, Positioned, etc.

Unfortunately the MD spec doesn't provide a unique name for the left and right sides. There's this:
![primary](https://cloud.githubusercontent.com/assets/1377460/13293280/e4197542-dad3-11e5-9e2b-1e37856ae763.png)
and this:
![left](https://cloud.githubusercontent.com/assets/1377460/13293287/e94d02b8-dad3-11e5-853e-6769985ff79f.png)

I suppose that we can refer to the left and right widgets however we want. Devs will have to learn what they mean.